atlp-rwanda/space-bn-frontend
# #175743401 ft(signup): add user signup page

#### What does this PR do?

- Implement user signup page

#### Description of Task to be completed? 

- Implement the signup form

- Implement mobile view of the page

- Write tests of the page

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`ft-signup-page-#175743401`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- Run **`npm test`** to run tests

- In your default browser navigate to **`http://localhost:3000/signup`**

> You should see a signup form 


#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker stories?

- [#175743401](https://www.pivotaltracker.com/n/projects/2476677/stories/175743401)

#### Screenshots (if appropriate)

<img width="1440" alt="Screen Shot 2020-12-02 at 10 19 56" src="https://user-images.githubusercontent.com/54619678/100849228-ee1de580-348a-11eb-9539-33e82a9b9892.png">


#### Questions:

- N/A
